docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -513,29 +513,36 @@ be the same for all subnational locations.
 2.5 Simulation scenarios
 ------------------------
 
-As of June, 2023, there are a total of 5 scenarios in the pregnancy simulation, :ref:`which can be found here <nutritionoptimizationpreg4.0>`. With the exception of the baseline scenario, all of the following child scenarios should be run on the outputs for each pregnancy scenario.
+As of June, 2023, there are a total of 5 scenarios in the pregnancy simulation, :ref:`which can be found here <nutritionoptimizationpreg4.0>`. With the exception of the baseline scenario, all of the following child scenarios should be run on the outputs for each pregnancy scenario unless otherwise noted, particularly for Wave III.
```

Comment:
The number of pregnancy scenarios has gotten a bit confusing, but we actually ran wave 2 with just 4...

0. Zero
1. IFA
2. MMS
3. MMS/BEP

There are two others we were considering: IFA/BEP, nothing/BEP.

We'll probably want to think through whether we want to include these extras or not, while will influence the total number of child scenarios scoped out here.